src/pages/Vote/index.tsx

```diff
@@ -11,8 +11,9 @@ import DelegateModal from 'components/vote/DelegateModal'
 import ProposalEmptyState from 'components/vote/ProposalEmptyState'
 import { useActiveWeb3React } from 'hooks/web3'
 import JSBI from 'jsbi'
+import CreateProposal from 'pages/CreateProposal'
```

Comment:
@moodysalem what would be the appropriate file structure for nested pages? As-is, they are all top-level files. Would it be ok to nest them under vote, ie renaming them to `pages/vote/index.ts`, `pages/vote/Governance.tsx`, `pages/vote/Vote.tsx`, `pages/vote/Propose.tsx`?